### PR TITLE
Add Social Recovery Module

### PR DESCRIPTION
**Pull request by nlordell**
This PR adds deployment information for the Social Recovery Module :). You can verify that the address matches the expected one from:

https://github.com/safe-global/safe-modules/blob/main/modules/recovery/CHANGELOG.md

### CODE REVIEW

File: src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json [hunk 1]
```diff
@@ -0,0 +1,721 @@
+{
+  "released": true,
+  "contractName": "FCLP256Verifier",
```
Comment (remedcu) [inline src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json:3]:
```suggestion
  "contractName": "SocialRecoveryModule",
```

Comment (nlordell) [reply src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json:3]: Oh wow! Can't believe I missed that. Thanks :)

File: src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json [hunk 2]
```diff
@@ -0,0 +1,721 @@
...
+  "version": "0.2.0",
```
Comment (remedcu) [inline src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json:4]:
```suggestion
  "version": "0.1.0",
```

File: src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json (hunk 1 above)
Comment (nlordell) [reply src/assets/safe-recovery-module/v0.1.0/social-recovery-module.json]: Version was wrong too :see_no_evil: - should be fixed now.

**Review by remedcu — APPROVED** [review]

**Pull request merged**